strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/providers/repository/RepositorySearchRequest.java

```diff
@@ -5,11 +5,13 @@
 
 public class RepositorySearchRequest
 {
+    public static final Integer MAX_LIMIT = 1000;
```

Comment:
@carlspring please note, we will limit search results for our safety.

Reply:
Why do we need this limit and what do you mean by "safety"?

Reply:
I mean that someone can make a search request for millions of results and it will cause OOM for us.

Reply:
Okay, that's what I thought.